aten/src/ATen/test/extension_backend_test.cpp

```diff
@@ -62,3 +64,12 @@ TEST(BackendExtensionTest, TestRegisterOp) {
         .aliasAnalysis(c10::AliasAnalysisKind::FROM_SCHEMA))
   );
 }
+
+void generic_override(const char* schema, torch::jit::Stack* stack) {
+  std::cerr << "arf\n";
+}
+
+TEST(BackendExtensionTest, TestBoxedFallback) {
+  globalATenDispatch().registerFallbackBoxedOp(TensorTypeId::XLATensorId, &generic_override);
```

Comment:
note: extension backends already switched away from `globalATenDispatch` to `torch::RegisterOperators`

Reply:
Yes this is a good idea.